include/usb/usb_common.h

```diff
@@ -1,6 +1,9 @@
-/***************************************************************************
+/*
+ * Copyright (c) 2017 PHYTEC Messtechnik GmbH
  *
+ * SPDX-License-Identifier: Apache-2.0
  *
+ * This file is derived from material that is
  * Copyright(c) 2015,2016 Intel Corporation.
  *
```

Comment:
I think you should put your copyright after Intel one since this is normal practice

Reply:
At least one cannot write that this is now "derived from material...", since the material is all still there, nothing visible added, nothing visible removed. In fact I think project policy has been that only substantially large rewrites of the code require a copyright to be added, the rest of the contributors are available with 'git shortlog' or 'git blame'.

Reply:
@finikorg Maybe we can do it differently, we could move all the standard descriptor relevant things to usb_descriptor.h, and then join usb_common.h and usbstruct.h.

>At least one cannot write that this is now "derived from material...", since the material is all still there, nothing visible added, nothing visible removed.

In fact b1f1fe4 is the fix for 98f0b86, and would be squashed with it.